static/app/views/organizationGroupDetails/groupReplays/groupReplays.tsx

```diff
@@ -0,0 +1,110 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import FeatureBadge from 'sentry/components/featureBadge';
+import PageHeading from 'sentry/components/pageHeading';
+import {PanelTable} from 'sentry/components/panels';
+import {t} from 'sentry/locale';
+import {PageContent, PageHeader} from 'sentry/styles/organization';
+import space from 'sentry/styles/space';
+import {NewQuery} from 'sentry/types';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
+import {Replay} from 'sentry/views/replays/types';
+
+import GroupReplaysTable from './groupReplaysTable';
+
+const DEFAULT_DISCOVER_LIMIT = 50;
+
+const GroupReplays = () => {
+  const location = useLocation();
+  const organization = useOrganization();
+  const params = useParams();
+
+  const getEventView = () => {
+    const {groupId} = params;
+    const eventQueryParams: NewQuery = {
+      id: '',
+      name: '',
+      version: 2,
+      fields: [
+        'eventID',
+        'project',
+        'timestamp',
+        'url',
+        'user.display',
+        'user.email',
+        'user.id',
+        'user.ip_address',
+        'user.name',
+        'user.username',
+      ],
+      projects: [],
+      orderby: '-timestamp',
+      query: `issue.id:${groupId}`,
+    };
+
+    return EventView.fromNewQueryWithLocation(eventQueryParams, location);
+  };
+
+  return (
+    <Fragment>
+      <StyledPageHeader>
+        <HeaderTitle>
+          {t('Replays')} <FeatureBadge type="alpha" />
+        </HeaderTitle>
+      </StyledPageHeader>
+      <StyledPageContent>
+        <DiscoverQuery
+          eventView={getEventView()}
+          location={location}
+          orgSlug={organization.slug}
+          limit={DEFAULT_DISCOVER_LIMIT}
+        >
+          {data => {
+            return (
+              <Fragment>
+                <StyledPanelTable
+                  isLoading={data.isLoading}
+                  isEmpty={data.tableData?.data.length === 0}
+                  headers={[t('Session'), t('Timestamp'), t('Duration'), t('Errors')]}
+                >
+                  {data.tableData ? (
+                    <GroupReplaysTable replayList={data.tableData.data as Replay[]} />
+                  ) : null}
+                </StyledPanelTable>
+              </Fragment>
+            );
+          }}
+        </DiscoverQuery>
+      </StyledPageContent>
+    </Fragment>
+  );
+};
+
+const StyledPageHeader = styled(PageHeader)`
+  background-color: ${p => p.theme.white};
```

Comment:
Should this be a theme alias? (you can quickly switch between dark/light mode by using cmd+shift+L)